Add documentation for PRs and Auto Insights (#2)
Codesee-io/docs-maps: docs/prs.md

# Pull Request Maps

> Once [a map is installed on a repository](./installation.md), pull request maps will be automatically generated for them.

Reviewer: ```suggestion
Once [a map is installed on a repository](./installation.md), Pull Request Maps will be automatically generated for every pull request made to the repository. 
```

> ## Generating a pull request map

Reviewer: ```suggestion
## Generating A Pull Request Map
```

> Might sound straightforward, and that's because it is! All you have to do is create a pull request on your repo.

Reviewer: ```suggestion
To get started, make a pull request to the repository.
```

> Once that's done, the `codesee-architecture-diagrams` bot will automatically comment on your pull request with an image of a map, along with a legend showing how these have changed. Below is an example from [Distribute Aid](https://distributeaid.org/)'s public repository:

Reviewer: ```suggestion
Once that's done, the `codesee-architecture-diagrams` bot will automatically comment on your pull request with an image of a Map showing the files impacted by the PR, along with a Map legend. Below is an example from [Distribute Aid](https://distributeaid.org/)'s public repository:
```
